types/chrome-remote-interface/index.d.ts

```diff
@@ -1,18 +1,51 @@
 // Type definitions for chrome-remote-interface 0.30
 // Project: https://github.com/cyrus-and/chrome-remote-interface
 // Definitions by: Khairul Azhar Kasmiran <https://github.com/kazarmy>
+//                 Seth Westphal <https://github.com/westy92>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // Minimum TypeScript Version: 3.8
 
 import type ProtocolProxyApi from 'devtools-protocol/types/protocol-proxy-api';
+import type ProtocolMappingApi from 'devtools-protocol/types/protocol-mapping';
 
 declare namespace CDP {
     interface BaseOptions {
         host?: string;
         port?: number;
+        secure?: boolean;
+        useHostName?: boolean;
+        alterPath?: (path: string) => string;
     }
 
-    interface ListJson {
+    interface Options extends BaseOptions {
+        target?: ((targets: Target[]) => Target | number) | Target | string;
+        protocol?: object;
```

Comment:
An example object appears to be https://github.com/cyrus-and/chrome-remote-interface/blob/v0.30.1/lib/protocol.json but this is probably not something to be worried about.

Reply:
Yeah, I think it's safe to keep it as a generic object for now; we can always add this in the future if desired.